modules/lang-painless/src/main/java/org/elasticsearch/painless/action/PainlessExecuteAction.java

```diff
@@ -105,7 +114,15 @@ private PainlessExecuteAction() {
         static final Map<String, ScriptContext<?>> SUPPORTED_CONTEXTS = Map.of(
                 "painless_test", PainlessTestScript.CONTEXT,
                 "filter", FilterScript.CONTEXT,
-                "score", ScoreScript.CONTEXT);
+                "score", ScoreScript.CONTEXT,
+                "boolean_script_field_script_field", BooleanFieldScript.CONTEXT,
```

Comment:
@nik9000 This "bug" with doubling the "script_field" suffix is unfortunate. WDYT about doing a hard break and fixing it? The only place the name is used is for stored scripts (but only in the api, to know which context to compile against) and in the node settings for script compilation limits, and with the latter it's not applicable here since unlimited is used.

Reply:
We disallowed stored scripts for runtime fields when we first built them and never re-allowed them. So I think that's mostly safe. I'm quite ok renaming things. I hadn't realized these names got out to users. Also - thing were different when we picked the names. We thought there'd be more different kinds of runtime fields. We never renamed them later. Anyway, what do you think of `boolean_field_script`? They are the script that runtime fields use for boolean fields *and* they are the script that we'll use to calculate boolean fields at index time. Same script context.

Reply:
I agree that it is odd to make users type this, but I would take this a step forward. I am not entirely comfortable even asking users to type context: "boolean_script_field" when they want to test a script for a boolean runtime field. It seems like there is a gap between how a runtime field is simulated and when it gets used in practice. There may be other ways to achieve this, but would it be a possibility to expose some artificial "runtime_field" context instead, and have another way to specify the type, for instance as part of the context setup? Any other ideas on this?